test/test_torch.py

```diff
@@ -8585,6 +8585,32 @@ def run_test(A_dims, b_dims, pivot=True):
         run_test((4, 4), (2, 1, 3, 4, 2))  # broadcasting A
         run_test((1, 3, 1, 4, 4), (2, 1, 3, 4, 5))  # broadcasting A & b
 
+    def test_dim_arg_reduction_scalar(self, device):
+        example = 4.0
+
+        types = [torch.double,
```

Comment:
You can use @dtypes, @dtypesIfCUDA decorators.

Reply:
Done, thanks!